packages/ui5-application-writer/test/defaults.test.ts

```diff
@@ -15,11 +15,11 @@ describe('Defaults', () => {
         ['1.80-snapshot', esmTypesVersionSince],
         ['metadata', typesVersionBest],
         [undefined, typesVersionBest],
-        ['1.109.1', typesVersionBest],
+        ['1.109.1', '~1.108.0'],
         [UI5_DEFAULT.TYPES_VERSION_BEST, typesVersionBest],
-        ['1.109-snapshot', typesVersionBest],
+        ['1.109-snapshot', '~1.108.0'],
```

Comment:
Doesn't `typesVersionBest` evaluate to the same value? Why replace the constant with the literal value?

Reply:
Done in e7992de982214fffc7fbc028ab8933b4530df2c6